samples/client/petstore/typescript-angular-v2/default/api/pet.service.ts

```diff
@@ -213,7 +213,11 @@ export class PetService {
         return httpParams;
     }
 
-    private addToHttpParamsRecursive(httpParams: HttpParams, value: any, key?: string): HttpParams {
+    private addToHttpParamsRecursive(httpParams: HttpParams, value?: any, key?: string): HttpParams {
+        if(value == null) {
```

Comment:
this won't match the template, a whitespace is missing

Reply:
The whitespace is missing when the sample code is generated :(

Reply:
To clarify, all I did in this commit was run `./bin/typescript-angular-petstore-all.sh` and the generated code was missing the whitespace. I believe the CI failed *because* I added the space manually previously.

Reply:
yes, you need to run `mvn clean package` before re-generating the samples

Reply:
...Oh. XD